cli/src/tasks/copy.ts

```diff
@@ -58,6 +61,7 @@ export async function copyCommand(
 export async function copy(
   config: Config,
   platformName: string,
+  inline: boolean,
```

Comment:
```suggestion
  inline: boolean = false,
```
If you set here the default value to false you don't need to edit multiple copy calls just to pass false.
Also, since we export the functions, there are possibilities of people using them in their tooling, so making the parameter mandatory can be considered a breaking change.

Same for copyCommand and the two sync equivalent methods were inline was introduced